website/source/docs/agent/config.html.md

```diff
@@ -67,6 +67,15 @@ client {
   }
 }
 
+consul {
+    # Consul's HTTP Address
+    address = "127.0.0.1:8500"
+
+    # Auto-registered Consul Service names
```

Comment:
I would document these parameters, but I wouldn't put them here in the configuration example.  This implies that someone may want to change this, but in practice they probably do not.